packages/model-viewer/src/three-components/Renderer.ts

```diff
@@ -34,14 +34,20 @@ export interface ContextLostEvent extends Event {
   sourceEvent: WebGLContextEvent;
 }
 
+// Between 0 and 1: larger means the average responds faster and is less smooth.
+const DURATION_DECAY = 0.2;
+const LOW_FRAME_DURATION = 18;   // ms
+const HIGH_FRAME_DURATION = 26;  // ms
```

Comment:
Just my personal preference, but I will usually suffix these kinds of constants with their unit in the name e.g., `HIGH_FRAME_DURATION_MS`.